<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QDialog" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>519</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <widget class="QWidget" name="background_widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>921</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background_widget {
background-color: qlineargradient(spread:pad, x1:0, y1:0.261, x2:1, y2:0.71,stop:0 rgba(64, 64, 64, 255), stop:1 rgba(158, 158, 158, 255));
}</string>
   </property>
   <widget class="QLabel" name="mainscreen_label">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>40</y>
      <width>450</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#mainscreen_label {
	font: 28pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Anime Mover and Renamer</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>681</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#start_button {
	border-radius: 15px;
	background-color: rgba(36, 36, 36, 255);
	color: white;
	font: 13pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="season_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#season_label{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>What season is this of the anime?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="season_input">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>138</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#season_input{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	background-color: rgba(0, 0, 0, 50);
	border-radius: 10px;
	border: 1px solid black;
	text-align: center;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="language_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#language_label{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Is the anime Dubbed or Subbed?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="language_input">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>138</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#language_input{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	background-color: rgba(0, 0, 0, 50);
	border-radius: 10px;
	border: 1px solid black;
	text-align: center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confirm_name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#confirm_name_label{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Is this name Correct?</string>
    </property>
   </widget>
   <widget class="QLabel" name="confirm_name_output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#confirm_name_output{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	background-color: rgba(0, 0, 0, 50);
	border-radius: 10px;
	border: 1px solid black;
	text-align: center;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="yes_button">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>205</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/circle.png</normaloff>../Icons/circle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="no_button">
    <property name="geometry">
     <rect>
      <x>785</x>
      <y>205</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Icons/circle.png</normaloff>../Icons/circle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="new_name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#new_name_label{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
}</string>
    </property>
    <property name="text">
     <string>Please enter the correct name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>421</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#back_button {
	border-radius: 15px;
	background-color: rgba(36, 36, 36, 255);
	color: white;
	font: 13pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_name_input">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#new_name_input{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	background-color: rgba(0, 0, 0, 50);
	border-radius: 10px;
	border: 1px solid black;
	text-align: center;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#error {
	color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="season_error">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#season_error {
	color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="language_error">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#language_error {
	color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
